jtrust-lib/src/main/java/be/fedict/trust/ocsp/OcspTrustLinker.java

```diff
@@ -120,8 +121,13 @@ public TrustLinkerResult hasTrustLink(final X509Certificate childCertificate,
 		}
 		LOG.debug("OCSP URI: " + ocspUri);
 
-		final OCSPResp ocspResp = this.ocspRepository.findOcspResponse(ocspUri,
-				childCertificate, certificate, validationDate);
+		OCSPResp ocspResp = null;
+		try {
+			ocspResp = this.ocspRepository.findOcspResponse(ocspUri, childCertificate, certificate, validationDate);
+		} catch (ServerNotAvailableException e) {
+			LOG.error("OCSP server is unavailable!", e);
+			throw new TrustLinkerResultException(TrustLinkerResultReason.OCSP_UNAVAILABLE, "OCSP server is unavailable!");
```

Comment:
1. Don't log and throw exceptions at the same time; better to log upstream where the exception is caught
2. Add the original exception `e` as a cause to the new `TrustLinkerResultException`